src/index.ts

```diff
@@ -353,7 +353,7 @@ export default function rehypePrettyCode(
 
         const lineNumbers: number[] = [];
         if (meta) {
-          const matches = meta.matchAll(/\{(.*?)\}/g);
+          const matches = meta.matchAll(/(?<!\S)\{(.*?)\}/g);
```

Comment:
Does Safari support this? I remember we had to use a trick to avoid negative lookbehinds to make it work on old Safari 😓

Reply:
Interesting, I didn't even think this would be running client-side. According to https://caniuse.com/js-regexp-lookbehind, it's been supported since Safari 16.4. That's also backed up by Safari's release notes: https://developer.apple.com/documentation/safari-release-notes/safari-16_4-release-notes

Reply:
If you're concerned about how recent that is, I could probably figure out another way?

Reply:
Depends how you're using it, but it can be used on either/both server and client yes. Safari 16.4 is fairly recent..

Reply:
Just pushed a new regex that avoids negative lookbehind by surrounding the pattern in `\B` (matches any non-word-boundary character). Strictly speaking only the leading `\B` is necessary to avoid picking up `showLineNumbers{number}`, but I figure you wouldn't want to match some theoretical `{number}word` meta property, either.